src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -8705,7 +8715,7 @@ public static ulong Splice(ulong[] first, ulong[] second, ulong[] maskArray, int
             return false;
         }
 
-        public static bool CheckLoadVectorFirstFaultingBehavior<TMem, TElem, TFault>(TMem[] firstOp, TElem[] result, Vector<TFault> faultResult)
```

Comment:
>CheckLoadVectorFirstFaultingBehavior [](http://example.com/codeflow?start=27&length=36)

fix the alignment here.

Reply:
I believe this comment doesn't reference the line you intended. Could you check?